nle/env/base.py

```diff
@@ -225,71 +319,7 @@ def __init__(
         # -1 so that it's 0-based on first reset
         self._episode = -1
 
-        space_dict = {
-            "glyphs": gym.spaces.Box(
-                low=0, high=nethack.MAX_GLYPH, **nethack.OBSERVATION_DESC["glyphs"]
-            ),
-            "chars": gym.spaces.Box(
-                low=0, high=255, **nethack.OBSERVATION_DESC["chars"]
-            ),
-            "colors": gym.spaces.Box(
-                low=0, high=15, **nethack.OBSERVATION_DESC["colors"]
-            ),
-            "specials": gym.spaces.Box(
-                low=0, high=255, **nethack.OBSERVATION_DESC["specials"]
-            ),
-            "blstats": gym.spaces.Box(
-                low=np.iinfo(np.int32).min,
-                high=np.iinfo(np.int32).max,
-                **nethack.OBSERVATION_DESC["blstats"],
-            ),
-            "message": gym.spaces.Box(
-                low=np.iinfo(np.uint8).min,
-                high=np.iinfo(np.uint8).max,
-                **nethack.OBSERVATION_DESC["message"],
-            ),
-            "program_state": gym.spaces.Box(
-                low=np.iinfo(np.int32).min,
-                high=np.iinfo(np.int32).max,
-                **nethack.OBSERVATION_DESC["program_state"],
-            ),
-            "internal": gym.spaces.Box(
-                low=np.iinfo(np.int32).min,
-                high=np.iinfo(np.int32).max,
-                **nethack.OBSERVATION_DESC["internal"],
-            ),
-            "inv_glyphs": gym.spaces.Box(
-                low=0,
-                high=nethack.MAX_GLYPH,
-                **nethack.OBSERVATION_DESC["inv_glyphs"],
-            ),
-            "inv_strs": gym.spaces.Box(
-                low=0, high=127, **nethack.OBSERVATION_DESC["inv_strs"]
-            ),
-            "inv_letters": gym.spaces.Box(
-                low=0, high=127, **nethack.OBSERVATION_DESC["inv_letters"]
-            ),
-            "inv_oclasses": gym.spaces.Box(
-                low=0,
-                high=nethack.MAXOCLASSES,
-                **nethack.OBSERVATION_DESC["inv_oclasses"],
-            ),
-            "screen_descriptions": gym.spaces.Box(
-                low=0, high=127, **nethack.OBSERVATION_DESC["screen_descriptions"]
-            ),
-            "tty_chars": gym.spaces.Box(
-                low=0, high=127, **nethack.OBSERVATION_DESC["tty_chars"]
-            ),
-            "tty_colors": gym.spaces.Box(
-                low=-15,
-                high=15,
-                **nethack.OBSERVATION_DESC["tty_colors"],
-            ),
-            "tty_cursor": gym.spaces.Box(
-                low=0, high=255, **nethack.OBSERVATION_DESC["tty_cursor"]
-            ),
-        }
-
+        space_dict = dict(NLE_SPACE_ITEMS) if space_dict is None else space_dict
```

Comment:
I think this is fine as the two liner

```suggestion
        if space_dict is None:
            space_dict = dict(NLE_SPACE_ITEMS)
```

Reply:
Wow, github allows committing suggestions.